Extensions/xcloud.js

```diff
@@ -1,7 +1,7 @@
 //* TITLE XCloud **//
-//* VERSION 0.2 REV B **//
+//* VERSION 0.3.0 **//
```

Comment:
Can we make the version `1.0.0` now?  No reason to keep it pre-1.0, especially given this is a rather major change.